lib/mongo/database/view.rb

```diff
@@ -47,6 +47,13 @@ class View
       #
       # @option options [ Integer ] :batch_size  The batch size for results
       #   returned from the listCollections command.
+      # @option options [ Hash ] :filter A filter on the collections returned.
+      # @option options [ true, false ] :authorized_collections A flag, when
+      #   set to true and used with nameOnly: true, that allows a user without the
```

Comment:
Since this method always adds `name_only`, the note about `nameOnly` does not apply to this method.